app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.build(comment_params)
+    @comment.user = current_user
+
+    return unless @comment.save
+
+    redirect_to user_post_path(@post.author, @post), notice: 'Comment created successfully'
+  end
```

Comment:
- [ ] @DevKaranJ Great work adding a method to add new comments,  however you have not used the `current_user` method as required in the project requirements.
![image](https://github.com/DevKaranJ/Blog/assets/61408860/69689a45-c3a7-4083-947c-ec93cc410ba3)